indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -56,7 +57,7 @@ def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
             for k in redis_docs:
                 redis_handler.set(k['_id'], k['values'])
 
-    def update(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
+    def update(self, keys: Iterable[str], values: Iterable[bytes], *args, **kwargs) -> None:
         """Update JSON-friendly serialized documents on the index.
 
         :param keys: document ids to update
```

Comment:
I think it needs a description of all params along with their types

Reply:
not needed as discussed